dts/bindings/auxdisplay/hit,hd44780.yaml

```diff
@@ -56,18 +56,18 @@ properties:
       Array of addresses for each row, will use defaults if not provided.
       Default is as per Hitachi HD44780 specification.
 
-  enable-line-rise-delay-us:
+  enable-line-rise-delay-ns:
     type: int
-    default: 800
+    default: 450
     description: |
-      Delay time (in us) to wait after enable line rises before setting low.
+      Delay time (in ns) to wait after enable line rises before setting low.
       Default is as per Hitachi HD44780 specification.
 
-  enable-line-fall-delay-us:
+  enable-line-fall-delay-ns:
     type: int
-    default: 100
+    default: 500
     description: |
-      Delay time (in us) to wait after enable line falls before sending
+      Delay time (in ns) to wait after enable line falls before sending
       another command. Default is as per Hitachi HD44780 specification.
```

Comment:
https://www.sparkfun.com/datasheets/LCD/HD44780.pdf page 49 this should be 1000

Reply:
Actually, the doc states that the whole cycle should take 1000ns. The whole cycle can be seen as a sum of 4 timings:  enable pulse width logical '1'(PWeh == 450ns), enable pulse rise time (ter == 25ns), enable pulse fall time (tef == 25ns) and enable line logical '0'  (which is not explicitly stated in the doc, but can be calculated from the previous parameters, and is 500ns as stated here). I think the driver shouldn't count on the enable pulse rise/fall timings, as this surely differs between different platforms, so I will change the value of this parameter to 550ns. Thanks for looking into it.